device_doctor/lib/src/ios_device.dart

```diff
@@ -76,6 +78,39 @@ class IosDeviceDiscovery implements DeviceDiscovery {
     }
     return healthCheckResult;
   }
+
+  Future<HealthCheckResult> certCheck({ProcessManager processManager}) async {
+    HealthCheckResult healthCheckResult;
+    try {
+      final String certCheckResult =
+          await eval('security', <String>['find-identity', '-p', 'codesigning', '-v'], processManager: processManager);
+      if (certCheckResult.contains('Apple Development: Flutter Devicelab') &&
```

Comment:
Would it make sense to validate using the certificate code rather the names?

Reply:
Do we have the case where certificate code is changed while we maintain the certificate name?